tensorflow/lite/micro/kernels/ethosu.cc

```diff
@@ -0,0 +1,18 @@
+#include "tensorflow/lite/c/common.h"
+
```

Comment:
Can you add a comment to this file explaining that this is a stub for non-Ethos platforms?

Reply:
Yes, will do.